assets/app/data/crowbar/services/crowbar-backup.factory.spec.js

```diff
@@ -36,7 +36,8 @@ describe('Crowbar Backup Factory', function () {
         describe('when create method is executed', function () {
 
             beforeEach(function () {
-                $httpBackend.expect('POST', '/utils/backups', undefined)
+                $httpBackend.expect('POST', '/api/upgrade/adminbackup', undefined, function (headers) {
+                    return headers.Accept == COMMON_API_V2_HEADERS.Accept })
```

Comment:
Please, use identity comparison here.
http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons